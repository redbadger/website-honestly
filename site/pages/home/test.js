import React from 'react';
import { shallow } from 'enzyme';
import Home from '.';
import AfterSignUp from './newsletter-slice/after-sign-up';
import BeforeSignUp from './newsletter-slice/before-sign-up';

describe('HomePage', () => {
  describe('Newsletter slice', () => {
    it('shows pre-signup content if the form has not been submitted yet - default state', () => {
      const wrapper = shallow(<Home data={{}} />);
      expect(wrapper.find(BeforeSignUp)).to.have.length(1);
    });

    it('changes "submitted" state when submitted', () => {
      const wrapper = shallow(<Home data={{}} />);
      wrapper.instance().onSubmit({
        preventDefault: () => {},
      });
      expect(wrapper.state('newsletterSubmitted')).to.equal(true);
    });

    it('shows post-signup content if the form has been submitted', () => {
      const wrapper = shallow(<Home data={{}} />);
      wrapper.setState({
        newsletterSubmitted: true,
      });
      expect(wrapper.find(AfterSignUp)).to.have.length(1);
    });
  });
});
