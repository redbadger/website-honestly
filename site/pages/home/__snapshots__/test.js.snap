// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`site/pages/home renders correctly 1`] = `
Array [
  <section
    class="homepageTopSlice"
    data-cy="top-slice"
  >
    <div
      class="sliceContainer"
    >
      <div
        class="sloganWrapper"
      >
        <h1
          class="badgerSlogan"
        >
          <span>
            Let’s make
          </span>
           
          <span>
            things better
          </span>
        </h1>
      </div>
      <p
        class="sloganDescription fadeInUp"
      >
        <a
          class="sloganLink"
          href="/what-we-do/"
        >
          We are digital transformation experts who 
          <span
            class="sloganUnderline"
          >
            innovate and 
          </span>
          <span
            class="lastWord"
          >
            <span
              class="sloganUnderline"
            >
              deliver.
            </span>
            <svg
              class="arrow"
              viewBox="0 0 22 22"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M11 1l10 10-10 10-1.754-1.754 6.959-7.018H1V9.772h15.205l-6.96-7.018z"
                fill="#FC1D42"
                fill-rule="evenodd"
                stroke="#FC1D42"
              />
            </svg>
          </span>
        </a>
      </p>
    </div>
  </section>,
  <div
    class="goldCoinsContainer"
    data-cy="gold-coins-slice"
  >
    <div
      class="goldCoinsContent"
    >
      <h2
        class="goldCoinsHeading"
      >
        See how we work
      </h2>
      <a
        class="goldCoinsCTA"
        data-link="experience-us-slice"
        href="/what-we-do/experience-us/"
      >
        Experience Red Badger
      </a>
    </div>
    <div
      class="goldCoinsVideoOverlay "
    />
    <div
      class="goldCoinsVideo "
    >
      
        
      <video
        autoplay="true"
        disablepictureinpicture="false"
        disableremoteplayback="false"
        id="gc-slice-video"
        loop="true"
        muted="true"
        src="HomepageVid.mp4"
      >
        
        
      </video>
    </div>
    <img
      alt="Awards"
      class="goldCoinsImage goldCoinsCurrentImage "
      sizes="(min-width: 1440px) 1440px, 100vw"
      src="Homepage_1.jpg"
      style="opacity:1"
    />
    <img
      alt="Awards"
      class="goldCoinsImage goldCoinsNextImage hide"
      sizes="(min-width: 1440px) 1440px, 100vw"
      src="Homepage_2.jpg"
    />
  </div>,
  <div
    class="WasmWhitepaperSliceWrapper"
    id="genz"
  >
    <div
      class="WasmWhitepaperSliceContent"
    >
      <div
        class="WasmWhitepaperSlice"
      >
        <div
          class="WasmWhitepaperSliceText"
        >
          <img
            alt="text reading: We Love Tech"
            class="WasmWhitepaperSliceTitleImage"
            src="title.png"
          />
          <div
            class="WasmWhitepaperSliceTextContent"
          >
            <div
              class="WasmWhitepaperSliceCTA"
            >
              <h5>
                DOWNLOAD THE WHITE PAPER
              </h5>
            </div>
            <div
              class="WasmWhitepaperSliceBodyheader"
            >
              <h3>
                Multi-cloud platforms are here:
              </h3>
            </div>
            <div
              class="WasmWhitepaperSliceBodySubHeader"
            >
              <h4>
                How WebAssembly, NATS and wasmCloud can move us beyond the cloud
              </h4>
            </div>
            <div>
              <a
                class="WasmWhitepaperSliceLink"
                href="https://bit.ly/393vkLn"
              >
                Download White Paper
              </a>
            </div>
          </div>
        </div>
        <div
          class="WasmWhitepaperSliceImageWrapper"
        >
          <img
            alt="An illustration of the WASM whitepaper"
            class="WasmWhitepaperSliceImage"
            src="whitepaper.png"
          />
        </div>
      </div>
    </div>
  </div>,
  <section
    class="caseStudyContainer"
    data-cy="case-study-slice"
    id="stats"
  >
    <h2
      class="heading"
    >
      We solve complex problems and deliver real impact.
    </h2>
    <div
      class="limitWidth"
    >
      <div
        class="figuresContainer"
      >
        <a
          class="figureLink"
          href="/our-work/case-study/fortnum-and-mason"
          title="Fortnum and mason case study"
        >
          <span
            class="statsContainer"
          >
            <span
              class="countdownText"
            >
              2,000
            </span>
            <span
              class="countdownCaseText"
            >
              days of innovation and continuous 
              <span
                class="lastWord"
              >
                deployment
                <div
                  class="arrow"
                >
                  <svg
                    viewBox="0 0 22 22"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M11 1l10 10-10 10-1.754-1.754 6.959-7.018H1V9.772h15.205l-6.96-7.018z"
                      fill="#FC1D42"
                      fill-rule="evenodd"
                      stroke="#FC1D42"
                    />
                  </svg>
                </div>
              </span>
            </span>
          </span>
          <span
            class="logoContainer"
          >
            <img
              alt="The logo of Fortnum & Masons"
              class="logo"
              sizes="(min-width: 980px) 288px, 180px"
              src="fortnum.png"
            />
          </span>
        </a>
        <a
          class="figureLink"
          href="our-work/case-study/retailer"
          title="Retailer case study"
        >
          <span
            class="statsContainer"
          >
            <span
              class="centerText"
            >
              Five months to clear an eight year 
              <span
                class="lastWord"
              >
                backlog
                <div
                  class="arrow"
                >
                  <svg
                    viewBox="0 0 22 22"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M11 1l10 10-10 10-1.754-1.754 6.959-7.018H1V9.772h15.205l-6.96-7.018z"
                      fill="#FC1D42"
                      fill-rule="evenodd"
                      stroke="#FC1D42"
                    />
                  </svg>
                </div>
              </span>
            </span>
          </span>
          <span
            class="logoContainer"
          >
            <span
              class="companyText"
            >
              Britain’s Biggest Retailer
            </span>
          </span>
        </a>
        <a
          class="figureLink"
          href="/our-work/case-study/financial-services-digital-transformation"
          title="Bank case study"
        >
          <span
            class="statsContainer"
          >
            <span
              class="centerText"
            >
              Catalysts for change, delivery deployment and 
              <span
                class="lastWord"
              >
                culture
                <div
                  class="arrow"
                >
                  <svg
                    viewBox="0 0 22 22"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M11 1l10 10-10 10-1.754-1.754 6.959-7.018H1V9.772h15.205l-6.96-7.018z"
                      fill="#FC1D42"
                      fill-rule="evenodd"
                      stroke="#FC1D42"
                    />
                  </svg>
                </div>
              </span>
            </span>
          </span>
          <span
            class="logoContainer"
          >
            <span
              class="companyText"
            >
              Giant Global Bank
            </span>
          </span>
        </a>
        <a
          class="figureLink"
          href="/our-work/case-study/financial-times"
          title="Financial times case study"
        >
          <span
            class="statsContainer"
          >
            <span
              class="countdownText"
            >
              30%
            </span>
            <span
              class="caseText"
            >
              uplift in reader 
              <span
                class="lastWord"
              >
                engagement
                <div
                  class="arrow"
                >
                  <svg
                    viewBox="0 0 22 22"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M11 1l10 10-10 10-1.754-1.754 6.959-7.018H1V9.772h15.205l-6.96-7.018z"
                      fill="#FC1D42"
                      fill-rule="evenodd"
                      stroke="#FC1D42"
                    />
                  </svg>
                </div>
              </span>
            </span>
          </span>
          <span
            class="logoContainer"
          >
            <img
              alt="The logo of the Financial Times"
              class="logo"
              sizes="(min-width: 980px) 270px, 169px"
              src="financialtimes.png"
            />
          </span>
        </a>
      </div>
      <div
        class="caseCompanies"
      >
        <div
          class="caseCompaniesLogo"
        >
          <img
            alt="The logo of Selfridges"
            src="selfridges.png"
          />
        </div>
        <div
          class="caseCompaniesLogo"
        >
          <img
            alt="The logo of Car Trawler"
            src="cartrawler.png"
          />
        </div>
        <div
          class="caseCompaniesLogo"
        >
          <img
            alt="The logo of the BBC"
            src="bbc.png"
          />
        </div>
        <div
          class="caseCompaniesLogo"
        >
          <img
            alt="The logo of Tesco"
            src="tesco.png"
          />
        </div>
        <div
          class="caseCompaniesLogo"
        >
          <img
            alt="The logo of the NHS"
            src="nhs.png"
          />
        </div>
      </div>
      <div
        class="buttonContainer"
      >
        <a
          class="button"
          data-cy="ourWorkButton"
          href="/our-work/"
        >
          See more of our work
        </a>
      </div>
    </div>
  </section>,
  <div
    class="mbpSlice"
  >
    <div
      class="mbpSlice__wrapper"
    >
      <div
        class="mbpSlice__bgShapes"
      >
        <img
          alt=""
          class="mbpSlice__shape01"
          src="shape01.png"
        />
        <img
          alt=""
          class="mbpSlice__shape02"
          src="shape02.png"
        />
        <img
          alt=""
          class="mbpSlice__shape03"
          src="shape03.png"
        />
        <img
          alt=""
          class="mbpSlice__shape04"
          src="shape04.png"
        />
        <img
          alt=""
          class="mbpSlice__shape05"
          src="shape05.png"
        />
      </div>
      <img
        alt=""
        class="mbpSlice__bgTriangle"
        src="bgTriangle.png"
      />
      <div
        class="mbpSlice__Content"
      >
        <svg
          data-name="Layer 1"
          id="prefix__Layer_1"
          viewBox="0 0 181 82.69"
        >
          <defs>
            <mask
              height="118.69"
              id="prefix__mask"
              maskUnits="userSpaceOnUse"
              width="180.54"
              x="0"
              y="0"
            >
              <g
                transform="translate(0 -.31)"
              >
                <g
                  id="prefix__mask-2"
                >
                  <path
                    d="M0 119h180.54V.31H0V119z"
                    fill="#fff"
                    fill-rule="evenodd"
                    id="prefix__path-1"
                  />
                </g>
              </g>
            </mask>
          </defs>
          <title>
            Mission_Beyond_Logo
          </title>
          <path
            class="prefix__cls-2"
            d="M38 82.69h143v-48H38v48z"
            id="prefix__Fill-1"
          />
          <g
            class="prefix__cls-3"
          >
            <path
              class="prefix__cls-2"
              d="M0 47.56h142.47V0H0v47.56z"
              id="prefix__Fill-17"
            />
          </g>
          <g
            class="prefix__cls-3"
          >
            <path
              class="prefix__cls-4"
              d="M32.65 18.61l-6.66 16.95h-2.2l-6.66-16.95v16.95H12.1V12h7.05l5.74 14.7L30.63 12h7.09v23.56h-5.07V18.61z"
              id="prefix__Fill-19"
            />
          </g>
          <g
            class="prefix__cls-3"
          >
            <path
              class="prefix__cls-4"
              d="M42.29 35.88h4.5V18.82h-4.5zm-.46-21.94a2.69 2.69 0 112.69 2.69 2.71 2.71 0 01-2.69-2.69z"
              id="prefix__Fill-20"
              transform="translate(0 -.31)"
            />
          </g>
          <g
            class="prefix__cls-3"
          >
            <path
              class="prefix__cls-4"
              d="M51.72 30.51a9.84 9.84 0 005.91 2.33c1.95 0 2.87-.67 2.87-1.73s-1.48-1.63-3.43-2c-2.94-.56-6.74-1.27-6.74-5.33 0-2.86 2.52-5.37 7-5.37a10.75 10.75 0 016.87 2.33l-1.75 3.06a7.2 7.2 0 00-5.06-2c-1.64 0-2.7.6-2.7 1.59S56 24.86 58 25.24c2.94.57 6.88 1.35 6.88 5.58 0 3.15-2.66 5.48-7.41 5.48a11.51 11.51 0 01-7.65-2.65z"
              id="prefix__Fill-21"
              transform="translate(0 -.31)"
            />
          </g>
          <g
            class="prefix__cls-3"
          >
            <path
              class="prefix__cls-4"
              d="M68.62 30.51a9.84 9.84 0 005.92 2.33c1.95 0 2.87-.67 2.87-1.73s-1.49-1.63-3.44-2c-2.94-.56-6.73-1.27-6.73-5.33 0-2.86 2.51-5.37 7-5.37a10.8 10.8 0 016.88 2.33l-1.76 3.06a7.22 7.22 0 00-5.07-2c-1.63 0-2.69.6-2.69 1.59s1.34 1.45 3.26 1.83c2.94.57 6.87 1.35 6.87 5.58 0 3.15-2.66 5.48-7.41 5.48a11.55 11.55 0 01-7.65-2.65z"
              id="prefix__Fill-22"
              transform="translate(0 -.31)"
            />
          </g>
          <g
            class="prefix__cls-3"
          >
            <path
              class="prefix__cls-4"
              d="M85.06 35.88h4.5V18.82h-4.5zm-.46-21.94a2.7 2.7 0 112.69 2.69 2.71 2.71 0 01-2.69-2.69z"
              id="prefix__Fill-23"
              transform="translate(0 -.31)"
            />
          </g>
          <g
            class="prefix__cls-3"
          >
            <path
              class="prefix__cls-4"
              d="M106.36 27.33c0-2.65-1.6-4.95-4.39-4.95s-4.36 2.3-4.36 4.95 1.56 5 4.36 5 4.39-2.3 4.39-5m-13.43 0a9.06 9.06 0 0118.11 0 9.06 9.06 0 01-18.11 0"
              id="prefix__Fill-24"
              transform="translate(0 -.31)"
            />
          </g>
          <g
            class="prefix__cls-3"
          >
            <path
              class="prefix__cls-4"
              d="M125.92 25.56c0-2.36-1.24-3.18-3.15-3.18a4.85 4.85 0 00-3.9 2v11.5h-4.5V18.82h4.5V21a7.84 7.84 0 016-2.65c3.75 0 5.56 2.12 5.56 5.44v12.09h-4.5z"
              id="prefix__Fill-25"
              transform="translate(0 -.31)"
            />
          </g>
          <g
            class="prefix__cls-3"
          >
            <path
              class="prefix__cls-4"
              d="M61.68 67.16c1.91 0 3.05-1.09 3.05-2.79a2.76 2.76 0 00-3.05-2.75H55.2v5.54zm-.18-9.82a2.6 2.6 0 100-5.19h-6.3v5.19zm-11.34-9.46H62.6c4.5 0 6.81 2.86 6.81 6a5.25 5.25 0 01-4.08 5.41 5.71 5.71 0 014.54 5.79c0 3.53-2.38 6.36-6.88 6.36H50.16z"
              id="prefix__Fill-26"
              transform="translate(0 -.31)"
            />
          </g>
          <g
            class="prefix__cls-3"
          >
            <path
              class="prefix__cls-4"
              d="M77.1 61.3h8.43a4 4 0 00-4.22-3.78 4.1 4.1 0 00-4.21 3.78m4.21-7.3c5.11 0 8.58 3.82 8.58 9.4v1H77.17a4.6 4.6 0 004.92 4 7 7 0 004.57-1.69l2 2.89a10.59 10.59 0 01-7.05 2.37c-5.21 0-9.15-3.5-9.15-9A8.7 8.7 0 0181.31 54"
              id="prefix__Fill-27"
              transform="translate(0 -.31)"
            />
          </g>
          <g
            class="prefix__cls-3"
          >
            <path
              class="prefix__cls-4"
              d="M94.64 74.37A2.26 2.26 0 0097 73.13l.64-1.45-7-17.3h4.82L100 66.25l4.46-11.87h4.86l-8 19.85c-1.28 3.21-3.55 4.06-6.49 4.13a12.48 12.48 0 01-2.23-.25l.67-4a4 4 0 001.42.28"
              id="prefix__Fill-28"
              transform="translate(0 -.31)"
            />
          </g>
          <g
            class="prefix__cls-3"
          >
            <path
              class="prefix__cls-4"
              d="M123.66 62.89c0-2.65-1.59-4.94-4.39-4.94s-4.36 2.29-4.36 4.94 1.56 5 4.36 5 4.39-2.3 4.39-5m-13.43 0a9.06 9.06 0 119 9 8.66 8.66 0 01-9-9"
              id="prefix__Fill-29"
              transform="translate(0 -.31)"
            />
          </g>
          <g
            class="prefix__cls-3"
          >
            <path
              class="prefix__cls-4"
              d="M143.22 61.12c0-2.36-1.24-3.17-3.15-3.17a4.86 4.86 0 00-3.9 2v11.49h-4.5V54.38h4.5v2.22a7.88 7.88 0 016-2.65c3.76 0 5.56 2.12 5.56 5.44v12h-4.5z"
              id="prefix__Fill-30"
              transform="translate(0 -.31)"
            />
          </g>
          <g
            class="prefix__cls-3"
          >
            <path
              class="prefix__cls-4"
              d="M164 59.89a4.94 4.94 0 00-3.94-1.89c-2.56 0-4.33 2-4.33 5s1.77 4.94 4.33 4.94a4.94 4.94 0 003.94-2.01zm0 9.39a6.71 6.71 0 01-5.28 2.58c-4.32 0-7.58-3.25-7.58-8.93s3.22-9 7.58-9a6.6 6.6 0 015.28 2.64v-8.69h4.54v23.56H164z"
              id="prefix__Fill-31"
              transform="translate(0 -.31)"
            />
          </g>
        </svg>
        <p>
          We’re on a mission to solve the world’s biggest challenges through radical collaboration.
        </p>
        <a
          href="https://missionbeyond.co.uk"
          rel="noopener noreferrer"
          target="_blank"
        >
          Find out more
        </a>
      </div>
    </div>
  </div>,
  <div
    class="testimonials"
    data-cy="testimonial-slice"
  >
    <div
      class="container"
    />
  </div>,
  <section
    class="techSlice withDivider"
    data-cy="tech-slice"
  >
    <div
      class="techSliceIntro"
    >
      <h2
        class="heading"
      >
        We love tech. But we only use what’s right for the job.
      </h2>
      <p
        class="projectsBlerb"
      >
        Here is a selection of what we’ve used on recent projects.
      </p>
    </div>
    <ul
      class="techList"
    >
      <li
        class="techListItem"
      >
        <figure
          class="techItem"
        >
          <img
            alt="React + React Native logo"
            class="techLogo"
            src="react_v1_x1.png"
            srcset="react_v1_x1.png 90w, react_v1.png 150w"
          />
          <figcaption
            class="techName"
          >
            React + React Native
          </figcaption>
        </figure>
      </li>
      <li
        class="techListItem"
      >
        <figure
          class="techItem"
        >
          <img
            alt="GraphQL logo"
            class="techLogo"
            src="graphql_v1_x1.png"
            srcset="graphql_v1_x1.png 90w, graphql_v1.png 150w"
          />
          <figcaption
            class="techName"
          >
            GraphQL
          </figcaption>
        </figure>
      </li>
      <li
        class="techListItem"
      >
        <figure
          class="techItem"
        >
          <img
            alt="Rust logo"
            class="techLogo"
            src="rust_v1_x1.png"
            srcset="rust_v1_x1.png 90w, rust_v1.png 150w"
          />
          <figcaption
            class="techName"
          >
            Rust
          </figcaption>
        </figure>
      </li>
      <li
        class="techListItem"
      >
        <figure
          class="techItem"
        >
          <img
            alt="Golang logo"
            class="techLogo--tall"
            src="golang_v1_x1.png"
            srcset="golang_v1_x1.png 90w, golang_v1.png 150w"
          />
          <figcaption
            class="techName"
          >
            Golang
          </figcaption>
        </figure>
      </li>
      <li
        class="techListItem"
      >
        <figure
          class="techItem"
        >
          <img
            alt="Typescript logo"
            class="techLogo"
            src="typescript_v1_x1.png"
            srcset="typescript_v1_x1.png 90w, typescript_v1.png 150w"
          />
          <figcaption
            class="techName"
          >
            Typescript
          </figcaption>
        </figure>
        <img
          alt=""
          class="techPlusSymbol"
          src="plus01_x2.png"
        />
      </li>
      <li
        class="techListItem"
      >
        <figure
          class="techItem"
        >
          <img
            alt="NodeJS logo"
            class="techLogo"
            src="nodejs_v1_x1.png"
            srcset="nodejs_v1_x1.png 90w, nodejs_v1.png 150w"
          />
          <figcaption
            class="techName"
          >
            NodeJS
          </figcaption>
        </figure>
      </li>
    </ul>
    <ul
      class="techList"
    >
      <li
        class="techListItem"
      >
        <figure
          class="techItem"
        >
          <img
            alt="AWS logo"
            class="techLogo--wide"
            src="AWS_v4_x1.png"
            srcset="AWS_v4_x1.png 90w, AWS_v4.png 150w"
          />
          <figcaption
            class="techName"
          >
            AWS
          </figcaption>
        </figure>
      </li>
      <li
        class="techListItem"
      >
        <figure
          class="techItem"
        >
          <img
            alt="GCP logo"
            class="techLogo"
            src="GCP_v1_x1.png"
            srcset="GCP_v1_x1.png 90w, GCP_v1.png 150w"
          />
          <figcaption
            class="techName"
          >
            GCP
          </figcaption>
        </figure>
      </li>
      <li
        class="techListItem"
      >
        <figure
          class="techItem"
        >
          <img
            alt="Azure logo"
            class="techLogo"
            src="azure_v1_x1.png"
            srcset="azure_v1_x1.png 90w, azure_v1.png 150w"
          />
          <figcaption
            class="techName"
          >
            Azure
          </figcaption>
        </figure>
      </li>
      <li
        class="techListItem"
      >
        <figure
          class="techItem"
        >
          <img
            alt="Docker logo"
            class="techLogo--wide"
            src="docker_v1_x1.png"
            srcset="docker_v1_x1.png 90w, docker_v1.png 150w"
          />
          <figcaption
            class="techName"
          >
            Docker
          </figcaption>
        </figure>
        <img
          alt=""
          class="techPlusSymbol"
          src="plus03_x2.png"
        />
      </li>
      <li
        class="techListItem"
      >
        <figure
          class="techItem"
        >
          <img
            alt="Kubernetes logo"
            class="techLogo"
            src="kubernetes_v1_x1.png"
            srcset="kubernetes_v1_x1.png 90w, kubernetes_v1.png 150w"
          />
          <figcaption
            class="techName"
          >
            Kubernetes
          </figcaption>
        </figure>
        <img
          alt=""
          class="techPlusSymbol"
          src="plus04_x2.png"
        />
      </li>
      <li
        class="techListItem"
      >
        <figure
          class="techItem"
        >
          <img
            alt="Istio logo"
            class="techLogo"
            src="istio_v1_x1.png"
            srcset="istio_v1_x1.png 90w, istio_v1.png 150w"
          />
          <figcaption
            class="techName"
          >
            Istio
          </figcaption>
        </figure>
      </li>
    </ul>
    <a
      class="moreBtn"
      href="/technology/"
    >
      More about the tech
    </a>
  </section>,
  <section
    class="contactUsContainer"
    data-cy="contact-us-slice"
    id="contactUs"
  >
    <h2
      class="header"
    >
      We can help you
    </h2>
    <div
      class="contentContainer"
    >
      <ul
        class="list"
      >
        <li
          class="item"
        >
          Create & validate new ideas
        </li>
        <li
          class="item"
        >
          Deliver great quality products & services, fast
        </li>
        <li
          class="item"
        >
          Be bold with technology
        </li>
        <li
          class="item"
        >
          Be more customer centric
        </li>
        <li
          class="item"
        >
          Improve efficiency with lean practices
        </li>
        <li
          class="item"
        >
          Build capability & confidence
        </li>
      </ul>
      <div
        class="imgContainer"
      />
    </div>
    <div>
      <a
        class="mailToLink"
        data-cy="email-link"
        href="mailto:hello@red-badger.com?Subject=Can%20you%20help%20me%20with%20..."
      >
        Send an email
      </a>
      <span
        class="talkToUs"
      >
        hello@red-badger.com
      </span>
    </div>
  </section>,
  <section
    class="newsletter"
    data-cy="newsletter-signup-slice"
  >
    <h2
      class="title"
    >
      Sign up to Badger News
      <span
        class="subTitle"
      >
        Be the first to hear about webinars, events, blogs and more.
      </span>
    </h2>
    <div
      class="hubspotBtn btnWrapper"
    >
      <span
        class="hs-cta-wrapper"
        id="hs-cta-wrapper-6a020b57-ac2f-4b6d-8273-503c7b261094"
      >
        <span
          class="hs-cta-node hs-cta-6a020b57-ac2f-4b6d-8273-503c7b261094"
          id="hs-cta-6a020b57-ac2f-4b6d-8273-503c7b261094"
        >
          <a
            data-cy="email-signup"
            href="https://cta-redirect.hubspot.com/cta/redirect/4210858/6a020b57-ac2f-4b6d-8273-503c7b261094"
            target="_blank"
          >
            <img
              alt="Sign Up"
              class="hs-cta-img"
              height="112"
              id="hs-cta-img-6a020b57-ac2f-4b6d-8273-503c7b261094"
              src="https://no-cache.hubspot.com/cta/default/4210858/6a020b57-ac2f-4b6d-8273-503c7b261094.png"
              style="border-width:0px;"
              width="249"
            />
          </a>
        </span>
        <script
          charset="utf-8"
          src="https://js.hscta.net/cta/current.js"
        />
        <script
          type="text/javascript"
        >
           hbspt.cta.load(4210858, '6a020b57-ac2f-4b6d-8273-503c7b261094', {}); 
        </script>
      </span>
    </div>
  </section>,
  <div
    class="awardsContainer"
    data-cy="awards-slice"
  >
    <div
      class="awards"
    >
      <img
        alt="Awards"
        class="awardsImage"
        sizes="(min-width: 1440px) 1440px, 100vw"
        src="awards-april-2018.jpg"
      />
    </div>
  </div>,
]
`;
